### PR TITLE
Craft multiple chickens at once, add JEI shift clicking support

### PR DESCRIPTION
**Pull request by comp500**
oh no

you could decrease the maximum bulk crafting count if 64 sounds like a bad idea.

### CODE REVIEW

**Comment by comp500** [comment]
Some other ideas I had:
- Automation (9 raw chicken nugget items on a crafting table produces a chicken, might lag with a lot of items though due to needing to check each entity)
- Tinker's Construct support (it already has a nugget cast)
    - This would allow you to make molten Chicken pipelines connected to Casting Tables for automated nugget distribution

**Comment by quat1024** [comment]
Lmao this is great

File: src/main/java/quaternary/chickennugget/CraftChickenRecipe.java [hunk 1]
```diff
@@ -14,6 +14,9 @@
 import net.minecraftforge.registries.IForgeRegistryEntry;
 
 public class CraftChickenRecipe extends IForgeRegistryEntry.Impl<IRecipe> implements IRecipe {
+	//Craft up to 64 chickens at once
+	public static final int MAXIMUM_CHICKENS_CRAFTED = 64;
```
Comment (juliand665) [inline src/main/java/quaternary/chickennugget/CraftChickenRecipe.java:18]: i don't think this should be public

Comment (quat1024) [reply src/main/java/quaternary/chickennugget/CraftChickenRecipe.java:18]: i dont think u should be public

Comment (juliand665) [reply src/main/java/quaternary/chickennugget/CraftChickenRecipe.java:18]: rip

Comment (comp500) [reply src/main/java/quaternary/chickennugget/CraftChickenRecipe.java:18]: oof

**Pull request merged**